applications/API/tests/test_forms.py

```
@@ -0,0 +1,288 @@
+    @classmethod
+    def tearDownClass(cls):
+        """Clean up class-level fixtures."""
+        super().tearDownClass()
+        # Add any class-level cleanup here if needed
+
+    def test_clean_anonymous_submission_valid(self):
+        """Test valid anonymous submission."""
+        form_data = self.base_form_data.copy()
+        form = InsultReviewForm(data=form_data)
+
+        self.assertTrue(form.is_valid())
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)